source/_docs/z-wave/controllers.markdown

```diff
@@ -12,15 +12,15 @@ You need to have a compatible Z-Wave adapter installed. The following devices ha
 
 {% warning %}
 
-The firmwares of 700 and 800 series Z-Wave adapters have several bugs which impact the stability of the mesh and can cause the adapter to become unresponsive. Because there is no known firmware version that is completely fixed, it is recommended to choose a firmware based on the following criteria:
+The firmwares of 700- and 800-series Z-Wave adapters have several bugs which impact the stability of the mesh and can cause the adapter to become unresponsive. Because there is no known firmware version that is completely fixed, it is recommended to choose a firmware based on the following criteria:
 
-- 700 series:
+- 700-series:
```

Comment:
Silicon Labs actually call it "700 series" and "800 series": https://www.silabs.com/wireless/z-wave/introduction-to-z-wave-800-series